deploy/aws/app/kube/README.md

```diff
@@ -14,7 +14,7 @@ No providers.
 
 | Name | Source | Version |
 |------|--------|---------|
-| <a name="module_app"></a> [app](#module\_app) | git::https://github.com/amido/stacks-terraform//aws/modules/infrastructure_modules/stacks_app | n/a |
+| <a name="module_app"></a> [app](#module\_app) | git::https://github.com/ensono/stacks-terraform//aws/modules/infrastructure_modules/stacks_app | n/a |
```

Comment:
Could you review this URL?  It doesn't seem to work either

Reply:
The amido one doesn't work either